README.md

```diff
@@ -104,17 +105,17 @@ Specify one or more irrigation zones here. The component calculates irrigation d
 - Bucket: You can manipulate this calculated value.
 - Lead Time: In seconds. Time needed to warm up your irrigation system, e.g. time to establish a connection or start a pump etc.
 
-You can Update all zones and Callculate all.
+You can update and callculate all automatic zones.
 
 **Per Zone Settings**
 You can change any value mentioned before. Additionaly there are some more options.
 
-- State: Wether Updating and Calculation of that zone should be enabled in mode 'Automatic' or 'Manual' or 'Disabled'
-- Module: Choose the calculation module (see below) shall be used for that zone to calculate irrigation duration
-- Mapping: Which sensor mapping (see below) shall be used for that zone
+- State: Wether updating and calculation of that zone should be enabled in mode 'Automatic' or 'Manual' or 'Disabled'.
+- Module: Choose the calculation module (see below) shall be used for that zone to calculate irrigation duration.
+- Mapping: Which sensor mapping (see below) shall be used for that zone.
 - Bucket: Either calculated or manualy set. Bucket >=0 no irrigation is necesarry, bucket <0 irrigation is necesarry.
-- Multiplier: Multiplies the duration of irrigation
-- Duration: Either calculated or manualy set
+- Multiplier: Multiplies the duration of the irrigation.
```

Comment:
... or divides if you do 0.5 for example :)

Reply:
I waited for your  formulation. :-D